containerized_ml_models/model_api_10/api.py

```diff
@@ -0,0 +1,24 @@
+import json
+import pickle
+from flask import Flask, request
+
+app = Flask(__name__)
+
+with open("model.pkl", "rb") as file:
+    pickle_in = pickle.load(file)
+
+
+@app.route("/ping", methods=["GET"])
+def ping():
+    return json.dumps({"ping": "success"})
```

Comment:
I recommend to use jsonify from Flask instead of json
See why it is better to jsonify: https://stackoverflow.com/questions/7907596/json-dumps-vs-flask-jsonify